### PR TITLE
🐛Fix LET Event Initialization; 🐛 Fix Getting Tasks in Dynamic UI

### LINKED ISSUE #1449: 🐛 LET Bug With More Active Correlations
#### Describe your issue

If you have started a process multiple times with the same correlation ID and try to work off the last started process with the LET,  the process does not get stopped. 

If you navigate to the LET over the dashboard view it works.

#### How can others reproduce the issue?

- start a process multiple times with the same correlation ID
- try to work off the process the last process 
- see that it does not get stopped

#### Possible solution

Fix LET

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [ ] I've checked if this issue already exists
- [ ] I've included all the information that I think is relevant
- [ ] I've added logs and/or screenshots (if applicable)
- [ ] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by SteffenKn**
## Changes

1. Fix Initialization of Events
2. Fix Getting Tasks in Dynamic UI

## Issues

Closes #1449 

PR: #1479 

### 1

- Run a diagram with a custom correlation ID
- Run the same diagram with the same custom correlation ID
- **Notice that the LET is still working correctly**

### 2

- Run a diagram with a user task with a custom correlation ID (Don't finish the user task)
- Run the same diagram with the same custom correlation ID
- Finish the user task
- **Notice that the user tasks will be finished in the LET**

### CODE REVIEW

File: src/modules/live-execution-tracker/live-execution-tracker.ts [hunk 1]
```diff
@@ -826,7 +827,8 @@ export class LiveExecutionTracker {
 
       await this._handleElementColorization();
 
-      const correlationIsActive: boolean = await isCorrelationActive();
+      const correlationIsActive: boolean = await isProcessInstanceActive();
```
Comment (ElRaptorus) [inline src/modules/live-execution-tracker/live-execution-tracker.ts]: For consistency:

```suggestion
      const processInstanceIsActive: boolean = await isProcessInstanceActive();
```

File: src/modules/live-execution-tracker/live-execution-tracker.ts [hunk 2]
```diff
@@ -826,7 +827,8 @@ export class LiveExecutionTracker {
...
+
       const correlationIsNotActive: boolean = correlationIsActive === false;
```
Comment (ElRaptorus) [inline src/modules/live-execution-tracker/live-execution-tracker.ts]:
```suggestion
      const processInstanceIsNotActive: boolean = processInstanceIsActive === false;
```

File: src/modules/live-execution-tracker/live-execution-tracker.ts [hunk 3]
```diff
@@ -826,7 +827,8 @@ export class LiveExecutionTracker {
...
       if (correlationIsNotActive) {
```
Comment (ElRaptorus) [inline src/modules/live-execution-tracker/live-execution-tracker.ts]:
```suggestion
      if (processInstanceIsNotActive) {
```

File: src/services/dynamic-ui-service/dynamic-ui.service.ts [hunk 4]
```diff
@@ -29,13 +29,11 @@ export class DynamicUiService implements IDynamicUiService {
...
-                           correlationId: string,
-                           processModelId: string,
+                           processInstanceId: string,
                            userTaskId: string): Promise<DataModels.UserTasks.UserTask> {
 
-    const userTaskList: DataModels.UserTasks.UserTaskList = await this._managementApiClient.getUserTasksForProcessModelInCorrelation(identity,
-                                                                                                                processModelId,
-                                                                                                                correlationId);
+    const userTaskList: DataModels.UserTasks.UserTaskList = await this._managementApiClient.getUserTasksForProcessInstance(identity,
+                                                                                                                processInstanceId);
```
Comment (ElRaptorus) [inline src/services/dynamic-ui-service/dynamic-ui.service.ts]: This breaking looks weird.

How about:

```ts
const userTaskList: DataModels.UserTasks.UserTaskList = 
  await this._managementApiClient.getUserTasksForProcessInstance(identity, processInstanceId);
```

File: src/services/dynamic-ui-service/dynamic-ui.service.ts [hunk 5]
```diff
@@ -54,13 +52,11 @@ export class DynamicUiService implements IDynamicUiService {
...
-                             correlationId: string,
-                             processModelId: string,
+                             processInstanceId: string,
                              manualTaskId: string): Promise<DataModels.ManualTasks.ManualTask> {
 
-    const manualTaskList: DataModels.ManualTasks.ManualTaskList = await this._managementApiClient.getManualTasksForProcessModelInCorrelation(identity,
-                                                                                                                      processModelId,
-                                                                                                                      correlationId);
+    const manualTaskList: DataModels.ManualTasks.ManualTaskList = await this._managementApiClient.getManualTasksForProcessInstance(identity,
+                                                                                                                                   processInstanceId);
```
Comment (ElRaptorus) [inline src/services/dynamic-ui-service/dynamic-ui.service.ts]: Same thing here:

```ts
const manualTaskList: DataModels.ManualTasks.ManualTaskList =
  await  this._managementApiClient.getManualTasksForProcessInstance(identity, processInstanceId);
```

File: src/modules/live-execution-tracker/live-execution-tracker.ts (hunk 1 above)
Comment (alexanderkasten) [inline src/modules/live-execution-tracker/live-execution-tracker.ts]:
```suggestion
      const processInstanceIsActive: boolean = await isProcessInstanceActive();
```

File: src/modules/live-execution-tracker/live-execution-tracker.ts (hunk 2 above)
Comment (alexanderkasten) [inline src/modules/live-execution-tracker/live-execution-tracker.ts]:
```suggestion
      const processInstanceIsNotActive: boolean = processInstanceIsActive === false;
```

File: src/modules/live-execution-tracker/live-execution-tracker.ts (hunk 3 above)
Comment (alexanderkasten) [inline src/modules/live-execution-tracker/live-execution-tracker.ts]:
```suggestion
      if (processInstanceIsNotActive) {
```

**Pull request merged**